remote-droid-guard/src/main/java/org/microg/gms/droidguard/DroidguardHelper.java

```diff
@@ -89,6 +90,31 @@
         return invoke(context, clazz, request.packageName, request.reason, response.byteCode.toByteArray(), request.androidIdLong, request.extras);
     }
 
+    public static int gmsVersionCode(String arch) {
+        switch (arch) {
+            case "arm64-v8a":
+                return 12688023;
+            case "x86":
+                return 12688026;
+            case "x86_64":
+                return 12688027;
+            default:
+                return 12688019;
+        }
+    }
+
+    public static String gmsVersionNamePrefix(String arch) {
+        switch (arch) {
+            case "arm64-v8a":
+                return "12.6.88 (040400-{{cl}})";
+            case "x86":
+            case "x86_64":
```

Comment:
Shouldn't these two be different (at least according to https://i.imgur.com/5oAWf02.png)

Reply:
Haven't done in-depth checks either but assuming the string that is sent is the one containing `{{cl}}` and not the actual version string, the x86_64 binary contains `12.6.88 (040700-{{cl}})` and no occurrence of `12.6.88 (040800-{{cl}})`. Maybe both versions share the same droidguard VM?